bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoderTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.bigtable.dataflow;
+
+import static org.apache.hadoop.hbase.util.Bytes.toBytes;
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.bigtable.dataflow.HBaseMutationCoder;
+import com.google.cloud.dataflow.sdk.coders.CoderException;
+
+import org.apache.hadoop.hbase.client.Delete;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.Put;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * Tests for {@link HBaseMutationCoder}.
+ */
+public class HBaseMutationCoderTest {
```

Comment:
Look at (CoderProperties)[https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/e872e0b55b131df432bdf3d34879a78dfdbba1c0/sdk/src/main/java/com/google/cloud/dataflow/sdk/testing/CoderProperties.java]

for helpers to automatically test coders.